mava/utils/config_utils.py

```diff
@@ -0,0 +1,41 @@
+# python3
+# Copyright 2021 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility functions for Mava config"""
+
+from collections.abc import MutableMapping
+from typing import List
+
+
+def flatten_dict(
```

Comment:
Is there a test for this? If not, I am happy to write one :)

Reply:
No test, please feel free to write one @RuanJohn! 🙌

Reply:
No test, if you are keen, please write one! 🙌

Reply:
On it :)

Reply:
Sorted :call_me_hand:

Reply:
Thanks 👍